subseas_flex_fcst/config-sample.yaml

```diff
@@ -12,10 +12,15 @@ loglevel: info
 # Forecast options
 y_transform: true
 forecast_mu_file: CFSv2_SubXPRCP_CCAFCST_mu_Apr_Apr-1-2022_wk1.txt
```

Comment:
Remove these old config options that are no longer used.

Reply:
That very one is used by layout for its initialization. But we can probably replace what is in the layout with something that would just pick the first mu file of the list. Just need to read one.